manifests/k8s-checkup-framework.yaml

```diff
@@ -7,30 +7,61 @@ metadata:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: checkup-launcher-sa
+  name: checkup-framework-sa
   namespace: k8s-checkup-framework
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
 metadata:
-  name: configmap-reader
-  namespace: k8s-checkup-framework
+  name: checkup-framework
 rules:
-  - apiGroups: [ "" ]
-    resources: [ "configmaps" ]
-    verbs: [ "get" ]
+- apiGroups: [ "" ]
```

Comment:
Please compare this file to the same file on PR #11

Reply:
The `configmaps` part is moved to line 31